fix: add support for endpoint prefix (#1035)
mcollovati/quarkus-hilla: README.md

more streamlined updates and improved stability. By leveraging the Vaadin Quarkus extension, users of `quarkus-hilla`
will benefit from enhanced compatibility with future Vaadin features.

### Custom Endpoint Prefix

> A custom endpoint prefix can be configured setting the `vaadin.endpoint.prefix` entry in `application.properties`.
> The extension will create a custom `connect-client.ts` file in the `frontend` folder, constructing the `ConnectClient`
> object with the configured prefix.

Reviewer: ```suggestion
A custom endpoint prefix can be configured by setting the `vaadin.endpoint.prefix` entry in `application.properties`. The extension will create a custom `connect-client.ts` file in the `frontend` folder and construct the `ConnectClient` object with the configured prefix.
```

> The `connect-client.ts` already exists and does not match the Hilla default template, it will not be overwritten.

Reviewer: ```suggestion
If `connect-client.ts` exists and does not match the default Hilla template, it is not overwritten.
```
